packages/plugin-phone/test/integration/spec/call.js

```diff
@@ -317,18 +576,135 @@ describe(`plugin-phone`, function() {
       it(`accepts a "includeLogs" boolean`);
     });
 
-    describe(`on(ringing)`, () => {
-      it(`gets triggered when the remote party acknowledges the call`);
-    });
+    describe(`triggered events`, () => {
+      describe(`connection events`, () => {
+        let call;
+        let triggerSpy;
+        beforeEach(() => {
+          call = spock.spark.phone.dial(mccoy.email);
+          triggerSpy = sinon.spy(call, `trigger`);
+          assert.equal(call.status, `initiated`);
+        });
 
-    describe(`on(connected)`, () => {
-      it(`gets triggered when the call is connected`);
-    });
+        describe(`on(ringing)`, () => {
+          it(`gets triggered when the remote party acknowledges the call`, () => {
+            return Promise.all([
+              mccoy.spark.phone.when(`call:incoming`)
+                .then(([c]) => {
+                  c.acknowledge();
+                  c.answer();
+                }),
+              call.when(`ringing`)
+                .then(() => assert.calledWith(triggerSpy, `ringing`)),
+              call.when(`connected`)
+                .then(() => call.hangup())
+            ]);
+          });
+        });
+
+        describe(`on(connected)`, () => {
+          it(`gets triggered when the call is connected`, () => {
+            return Promise.all([
+              mccoy.spark.phone.when(`call:incoming`)
+                .then(([c]) => {
+                  c.acknowledge();
+                  c.answer();
+                }),
+              call.when(`connected`)
+                .then(() => assert.calledWith(triggerSpy, `connected`))
+                .then(() => call.hangup())
+            ]);
+          });
+        });
+
+        describe(`on(disconnected)`, () => {
+          it(`gets triggered when the call is disconnected`, () => {
+            return Promise.all([
+              mccoy.spark.phone.when(`call:incoming`)
+                .then(([c]) => {
+                  c.acknowledge();
+                  c.answer();
+                }),
+              call.when(`connected`)
+                .then(() => call.hangup()),
+              call.when(`disconnected`)
+                .then(() => assert.calledWith(triggerSpy, `disconnected`))
+            ]);
+          });
+        });
+      });
+
+      describe(`in-progress events`, () => {
+        let call;
+        beforeEach(() => {
+          call = spock.spark.phone.dial(mccoy.email);
+          assert.equal(call.status, `initiated`);
+        });
+
+        describe(`on(remoteAudioMuted:change)`, () => {
+          // stopSendingAudio doesn't trigger this change but should
```

Comment:
can you change the comment to something more like "when the remote part invokes stop sending audio, the local party doesn't receive the update"? I had to stare at it for a while before I realized the comment was, in fact, valid.